tests/snuba/search/test_backend.py

```diff
@@ -40,7 +40,7 @@ def setUp(self):
             data={
                 "fingerprint": ["put-me-in-group1"],
                 "event_id": "a" * 32,
-                "message": "foo",
+                "message": "This is a really long test message. You can ignore it and skip right to the end, where you will find the only relevant piece of this message. It is a 3 letter piece and it begins with an f. This is a really long test message. You can ignore it and skip right to the end, where you will find the only relevant piece of this message. It is a 3 letter piece and it begins with an f. This is a really long test message. You can ignore it and skip right to the end, where you will find the only relevant piece of this message. It is a 3 letter piece and it begins with an f. And your special word is: foo.",
```

Comment:
Just confirming, this is the test to make sure that a long message works correctly? It might be worth having an explicit test especially for this case.

Reply:
I originally did that under the assumption that messages in Postgres were cut off after 98 characters, which doesn't seem to be true...and I think the limiting factor is nodestore limiting messages to 8KB?

I could be wrong.

Reply:
Hmm, there's definitely truncation that looks like it's less thank 8k if you look at rows in redash, like the one mentioned in the ticket:
```
select message, * 
from sentry_groupedmessage sg 
where id = 1171934479;
```
That one fails to fetch if you search for it in issue search, but succeeds in event search. The truncation there looks like it might be 256 characters. No idea where it's happening though.

Reply:
You're definitely right that truncation happens at 256 somewhere. Maybe that's the max length of the field in Postgres?

Searching for that event with the long message FAILS when the search term is >256 chars into the string using `master`. It passes using this branch. So this test does ensure that we're using snuba for the message search and no longer Postgres.

I'll write up a separate, more explicit, test for this.